Assets/Plugins/Clean/Tests/ComponentPropertiesTests.cs

```diff
@@ -0,0 +1,36 @@
+using NUnit.Framework;
+using Plugins.Clean.Editor;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Plugins.Clean.Tests
+{
+    public class ComponentPropertiesTests
+    {
+        [Test]
+        public void ComponentProperties_Selectable_InputField()
+        {
+            var button = new GameObject().AddComponent<Button>();
+            button.transition = Selectable.Transition.Animation;
+            var copy = new ComponentProperties.SelectableObject(button);
+            
+            Assert.AreEqual(button.transition, Selectable.Transition.Animation);
+        }
+        
+        [Test]
+        public void ComponentProperties_Text_CopiesText()
+        {
+            var go1 = new GameObject();
+            var text = go1.AddComponent<Text>();
+            text.text = "testing123";
+            var copy = new ComponentProperties.Text(text);
+
+            var go2 = new GameObject();
+            var tmp = go2.AddComponent<TextMeshProUGUI>();
+            copy.Apply(tmp);
+        
+            Assert.AreEqual("testing123", tmp.text);
```

Comment:
Should we test for more than just the text? I mean, there's probably text alignment, fontsize etc... that we can also check?

Reply:
Yep, could test every property.  Probably should before we use it in production!